src/layout/dom.ts

```diff
@@ -1,9 +1,14 @@
+import { Priority } from "@clarity-types/core";
 import { Constant, NodeChange, NodeInfo, NodeValue, Source } from "@clarity-types/layout";
 import config from "@src/core/config";
+import { schedule } from "@src/core/task";
 import time from "@src/core/time";
+import discover from "@src/layout/discover";
 import selector from "@src/layout/selector";
 
 let index: number = 1;
+const BLACKLIST_TYPES = ["password", "hidden", "email"];
```

Comment:
This list might need to be the same list as you have for name. See : [https://www.w3schools.com/html/html_form_input_types.asp](https://www.w3schools.com/html/html_form_input_types.asp)
Unclear if the name should be used here at all.

Reply:
I've added tel.